nilearn/decoding/tests/test_same_api.py

```diff
@@ -120,42 +138,56 @@ def test_graph_net_and_tvl1_same_for_pure_l1(max_iter=100, decimal=2):
 
     # results should be exactly the same for pure lasso
     a = tvl1_solver(
-        unmasked_X, y, alpha, 1.0, mask, loss="mse", max_iter=max_iter
+        unmasked_X,
+        y,
+        alpha,
+        1.0,
+        mask,
+        loss="mse",
+        max_iter=max_iter,
+        verbose=0,
     )[0]
     b = _graph_net_squared_loss(
-        unmasked_X, y, alpha, 1.0, max_iter=max_iter, mask=mask
+        unmasked_X, y, alpha, 1.0, max_iter=max_iter, mask=mask, verbose=0
     )[0]
 
-    mask = nibabel.Nifti1Image(mask.astype(np.float64), np.eye(4))
-    X = nibabel.Nifti1Image(X.astype(np.float64), np.eye(4))
-    for standardize in [True, False]:
-        sl = BaseSpaceNet(
-            alphas=alpha,
-            l1_ratios=1.0,
-            mask=mask,
-            penalty="graph-net",
-            max_iter=max_iter,
-            standardize=standardize,
-        ).fit(X, y)
-        tvl1 = BaseSpaceNet(
-            alphas=alpha,
-            l1_ratios=1.0,
-            mask=mask,
-            penalty="tv-l1",
-            max_iter=max_iter,
-            standardize=standardize,
-        ).fit(X, y)
-
-        # Should be exactly the same (except for numerical errors).
-        # However because of the TV-L1 prox approx, results might be 'slightly'
-        # different.
-        np.testing.assert_array_almost_equal(a, b, decimal=decimal)
-        np.testing.assert_array_almost_equal(
-            sl.coef_, tvl1.coef_, decimal=decimal
-        )
-
-
-def test_graph_net_and_tvl1_same_for_pure_l1_logistic(max_iter=20, decimal=2):
+    # Should be exactly the same (except for numerical errors).
```

Comment:
Maybe remove (or merge) the comment just above saying that results should be exactly the same.